docsk8s/troubleshooting/ls-k8s-common-problems.asciidoc

```diff
@@ -1,25 +1,59 @@
 [[ls-k8s-common-problems]]
 === Common problems
 
-Following are some suggested resolutions to problems that you may encounter when running Logstash in a Kubernetes environment.
+Following are some suggested resolutions to problems that you may encounter when running {ls} in a Kubernetes environment.
 
-* <<problem-01>>
-* <<problem-02>>
+* <<problem-keep-restart>>
+* <<problem-oom>>
 
 [float]
-[[problem-01]]
+[[problem-keep-restart]]
 === Logstash keeps restarting
 When you check the running Kubernetes pods status, {ls} shows continual restarts. 
 
-This can be caused by...
+[source,bash]
+--
+NAMESPACE     NAME                                  READY   STATUS    RESTARTS      AGE
+default       logstash-f7768c66d-grzbj              0/1     Running   3 (55s ago)   6m32s
+--
 
-To resolve the problem...
+This can be caused by a few issues:
 
 [float]
-[[problem-02]]
-=== Logstash stops with an OOM errors
-The {ls} pod continues to stop running due to an out of memory (OOM) error.
+[[problem-nometric]]
+==== Metrics API not accessible to `readinessProbe`
 
-This can be caused by...
+If the `readinessProbe` is unable to access the health check endpoint, the {ls} process will be continuously stopped and restarted. To fix it, set the following in `logstash.yml` in `ConfigMap`.
 
-To resolve the problem...
+[source,bash]
+--
+api.http.host: 0.0.0.0
+--
+
+[float]
+[[problem-delay]]
+==== {ls} startup process takes longer than `initialDelaySeconds`
+
+It may be necessary to review the time constraints of `readinessProbe` and `livenessProbe` to ensure that {ls} has enough time to start up and expose the health check endpoint for the `readiness` and `liveness` probes to access.
```

Comment:
```suggestion
Review the time constraints of `readinessProbe` and `livenessProbe` to ensure that {ls} has enough time to start up and expose the health check endpoint for the `readiness` and `liveness` probes to access.
```

Reply:
It's best when we speak directly to a user. In cases where we don't want to be too prescriptive, we can use "Consider <whatever it is they should try.>."